src/constraint_solver/io.cc

```diff
@@ -2291,13 +2310,28 @@ bool CpModelLoader::BuildFromProto(const CpSequenceVariable& proto) {
   return true;
 }
 
+size_t CpModelLoader::IntegerExpressionCount() const {
+  return expressions_.size();    
+}
+
 IntExpr* CpModelLoader::IntegerExpression(int index) const {
   CHECK_GE(index, 0);
   CHECK_LT(index, expressions_.size());
   CHECK(expressions_[index] != nullptr);
   return expressions_[index];
 }
 
+IntExpr* CpModelLoader::IntegerExpressionByName(const std::string& name) const {
```

Comment:
I do not want to store names in a hash map. This can be done at the client level.